src/Templates/ProjectTemplates/VisualBasic/VisualBasicNetStandardClassLibrary/VisualBasicNetStandardClassLibrary.csproj

```diff
@@ -53,7 +53,6 @@
   <ItemGroup>
     <VSTemplate Include="ClassLibrary.vstemplate">
       <SubType>Designer</SubType>
-      <OutputSubPath>.NET Standard</OutputSubPath>
```

Comment:
Isnt this line the only change you need to remove the node in the UI? I like having the folder in our repo to organize it better.